azure-pipelines.yml

```diff
@@ -35,6 +39,7 @@ jobs:
     enablePublishBuildArtifacts: true
```

Comment:
What is the difference between enablePublishBuildArtifacts and enablePublishBuildAssets, and could these be infered as 'true' when enablePublishUsingPipelines = true?

Reply:
Right now the goal was to make async publishing opt in and easy to turn it on/off, so we added `enabledPublishUsingPipelines`. Once this form of publishing proves stable we'll remove that parameter. AFAIU `enablePublishBuildAssets` is for publishing to BAR/Maestro. `enablePublishBuildArtifacts` is to publish logs to AzDO Artifacts and seems to be orthogonal to the others.